operations/mimir-mixin/alerts/blocks.libsonnet

```diff
@@ -139,10 +139,31 @@
         {
           alert: $.alertName('IngesterTSDBWALCorrupted'),
           expr: |||
-            rate(cortex_ingester_tsdb_wal_corruptions_total[5m]) > 0
-          |||,
+            # alert when there are more than one corruptions
+            count by (%(per_namespace_label)s) (rate(cortex_ingester_tsdb_wal_corruptions_total[5m]) > 0) > 1
```

Comment:
I think you should use `alert_aggregation_labels` instead of `per_namespace_label`.

_Same comment applies below._

Reply:
pushed in https://github.com/grafana/mimir/pull/4920/commits/da208e834109f3e71a823502e13c371785a1f047